Npcs/SpriggitMudcrab.yaml

```diff
@@ -23,8 +23,8 @@ Configuration:
 Template: 02183E:Skyrim.esm
 Race: 109C7C:Skyrim.esm
 AIData:
-  Confidence: Average
-  EnergyLevel: 50
+  Confidence: Brave
+  EnergyLevel: 55
```

Comment:
maybe more, like 60?

Reply:
lol

Reply:
why are you so anti-mudcrabs, make it > 9000